web/libs/datamanager/src/components/DataGroups/ImageDataGroup.jsx

```diff
@@ -7,12 +7,14 @@ export const ImageDataGroup = (column) => {
     value,
     original,
     field: { alias },
+    columnCount,
   } = column;
   const root = getRoot(original);
+  const imageHeight = ImageDataGroup.height * Math.max(1, 8 - columnCount);
```

Comment:
this is a little bit nit-picky but if we can define 8 as some sort of constant to help define what it is that would be great

Reply:
ah, yes, forgot to leave this comment! you are actually using 7 in another place, which is a very good example of issues without constanst like this